Engine/AI/NavMesh.h

```diff
@@ -14,7 +14,7 @@
 class PanelNavMesh;
 class dtNavMeshQuery;
 class dtNavMesh;
-class dtMeshTile;
+struct dtMeshTile;
```

Comment:
Why changed dtMeshTile from a class to struct? What is the improvement?

Reply:
This forward declaration was incorrect, because `dtMeshTile` is a struct. So I changed it to avoid a warning